### ISSUE TITLE
Redundant information when summarizing traces

### ISSUE THREAD

**Issue by Jasonleng** [lnccbrown/HSSM#216]
When I fit a hierarchical model and use arviz.summary to inspect the traces, I will get a lot of entries in the summary table and it seems like many of the rows are redundant information. Probably because of that the summary function will take a long time to finish:
<img width="354" alt="Screen Shot 2023-07-19 at 10 21 49 AM" src="https://github.com/lnccbrown/HSSM/assets/7433093/1c8396fd-523b-4f92-83e0-fb014d062b40">
Not sure if possible to remove these rows.

**Comment by frankmj** [comment]
I noticed the same thing and I think this is also related to [#211](https://github.com/lnccbrown/HSSM/issues/211)

**Comment by digicosmos86** [comment]
Yeah that does look weird. Can you let us know your model specification? Thanks!

**Comment by Jasonleng** [comment]
> 

This redundancy does not always happen. If I run a model with just the random structure for one of the parameter like this:
`my_hssm = hssm.HSSM(
    data=sub_data,
    model="race_2",
    loglik='test2.onnx',
    loglik_kind='approx_differentiable',
    model_config={
        "list_params":['v0','v1','a','t'],
        "bounds":{'v0':(0,1),
                  'v1':(0,4),
                  'a':(1,3),
                  't':(0.1,2)}},
    include=[
        {
            "name": "v0",
            "prior": {
                "Intercept": {
                    "name": "Uniform",
                    "lower": 0,
                    "upper": 0.2,
                    "initval": 0.1,
                },
            },
            "formula": "v0 ~ 1 + (1|subj_idx)",
            "link": "identity",
        },
    
    ],
)`
I got this without any redundant print out:
<img width="514" alt="Screen Shot 2023-07-20 at 9 07 41 AM" src="https://github.com/lnccbrown/HSSM/assets/7433093/97f33319-c208-4ef3-ad46-0178dbb37906">
However, when I run a model with random structure for the other parameters, like this:
`my_hssm = hssm.HSSM(
    data=sub_data,
    model="race_2",
    loglik='test2.onnx',
    loglik_kind='approx_differentiable',
    model_config={
        "list_params":['v0','v1','a','t'],
        "bounds":{'v0':(0,1),
                  'v1':(0,4),
                  'a':(1,3),
                  't':(0.1,2)}},
    include=[
        {
            "name": "v0",
            "prior": {
                "Intercept": {
                    "name": "Uniform",
                    "lower": 0,
                    "upper": 0.2,
                    "initval": 0.1,
                },
            },
            "formula": "v0 ~ 1 + (1|subj_idx)",
            "link": "identity",
        },
                {
            "name": "v1",
            "prior": {
                "Intercept": {
                    "name": "Uniform",
                    "lower": 0,
                    "upper": 4,
                    "initval": 2.0,
                },
            },
            "formula": "v1 ~ 1 + (1|subj_idx)",
            "link": "identity",
        },
        {
            "name": "a",
            "prior": {
                "Intercept": {
                    "name": "Uniform",
                    "lower": 1.0,
                    "upper": 3.0,
                    "initval": 2.0,
                },
            },
            "formula": "a ~ 1 + (1|subj_idx)",
            "link": "identity",
        },
        {
            "name": "t",
            "prior": {
                "Intercept": {
                    "name": "Uniform",
                    "lower": .001,
                    "upper": 1.0,
                    "initval": 0.5,
                },
            },
            "formula": "t ~ 1 + (1|subj_idx)",
            "link": "identity",
        }
    
    ],
)`
It will produce the redundant printout.

**Comment by AlexanderFengler** [comment]
This is bambi behavior (which we are in talks about fixing), but not specifically an error. 
What happens is that the deterministics (trial by trial parameters which are regression outcomes) are included in the `.trace`  object and the plot doesn't know which to include or exclude. 
You can use the `var_names` argument in the plot and pass e.g. `var_names=['~t']` to suppress the 'redundant' outputs. 

We could provide wrappers around the plot function that force a different default behavior instead, but hoping to make the behavior consistent across parameters on the bambi side (currently `v` is treated differently).

**Comment by digicosmos86** [comment]
Closed via #303
